src/promptflow/promptflow/_sdk/_constants.py

```diff
@@ -459,6 +459,12 @@ class LineRunFieldName:
     EVALUATIONS = "evaluations"
 
 
+class CreatedByFieldName:
```

Comment:
Do we expect runtime use these const? If expect runtime use these const, we may need to move them to promptflow._constants.